lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/plugin/Action.js

```diff
@@ -149,6 +167,12 @@ qx.Class.define("callbackery.ui.plugin.Action", {
                         }
                         if (btCfg.key){
                             this._buttonMap[btCfg.key]=button;
+                            if (btCfg.urlAction) {
+                                this._buttonUrlActionMap[btCfg.key] = {
+                                    button    : button,
+                                    urlAction : btCfg.urlAction
+                                };
```

Comment:
as discussed, I would just make this an array

Reply:
Done.